openedx/core/djangoapps/user_api/completion/views.py

```diff
@@ -0,0 +1,62 @@
+import csv
+import logging
+
+from rest_framework.authentication import SessionAuthentication
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
+from edx_rest_framework_extensions.auth.session.authentication import SessionAuthenticationAllowInactiveUser
+from rest_framework import permissions
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from .tasks import migrate_progress, OUTCOME_MIGRATED
+
+log = logging.getLogger(__name__)
+
+
+class MigrateProgressView(APIView):
+    """
+    Migrates user progress for a set of user pairs.
+    Only admins can use this.
+    """
+
+    authentication_classes = (
+        JwtAuthentication,
+        SessionAuthentication,
+        SessionAuthenticationAllowInactiveUser,
+    )
+
+    permission_classes = (
+        permissions.IsAuthenticated,
+        permissions.IsAdminUser,
+    )
+
+    def post(self, request):
+        """
+        POST /api/user/v1/completion/migrate/
```

Comment:
Can you improve this a bit and the required inputs? i.e. how should the request be and how should the CSV be?